src/xrpld/app/misc/WasmHostFunc.h

```diff
@@ -109,13 +110,13 @@ struct HostFunctions
         return beast::Journal{beast::Journal::getNullSink()};
     }
 
-    virtual Expected<std::uint32_t, HostFunctionError>
+    virtual Expected<int32_t, HostFunctionError>
```

Comment:
Why this change?

Reply:
@pwang200 and I discussed and since it won't reach `int32::max` for like 500 years, this will be much easier to work with and will align with` get_base_fee`

Reply:
[7d2343d](https://github.com/XRPLF/rippled/pull/5674/commits/7d2343d5771812db0d29c37906bbe6d16d32fe80)